src/main/java/org/openrewrite/kotlin/internal/KotlinParserVisitor.java

```diff
@@ -3451,6 +3467,9 @@ public J visitRegularClass(FirRegularClass regularClass, ExecutionContext ctx) {
                 jcEnums.add(declaration);
             } else if (declaration instanceof FirPrimaryConstructor) {
                 firPrimaryConstructor = (FirPrimaryConstructor) declaration;
+            } else if (declaration instanceof FirProperty && declaration.getSource().getKind() instanceof KtFakeSourceElementKind.PropertyFromParameter) {
```

Comment:
getSource on FirElements is nullable, so we need to check `declaration.getSource() != null`

Reply:
revised.